src/main/java/com/hoteldetails/pad/mappers/roomavailability/response/RoomAvailabilityResponseMapper.java

```diff
@@ -0,0 +1,48 @@
+package com.hoteldetails.pad.mappers.roomavailability.response;
+
+
+import com.hotel.service.common.ResponseStatus;
+import com.hotel.service.roomavailability.Rate;
+import com.hotel.service.roomavailability.RoomAvailabilityResponse;
+
+import org.opentravel.ota._2003._05.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.hotel.service.util.ProtoBufUtil.safeSetProtoField;
+import static com.hoteldetails.pad.util.ApiConstants.FAILURE;
+import static com.hoteldetails.pad.util.ApiConstants.SUCCESS;
+import static java.util.Objects.nonNull;
+
+@Component
+public class RoomAvailabilityResponseMapper {
+
+    @Autowired
+    RateMapper rateMapper;
+
+    public RoomAvailabilityResponse map(PropertyAvailabilityRS response) {
+        RoomAvailabilityResponse.Builder responseBuilder = RoomAvailabilityResponse.newBuilder();
+        if (nonNull(response.getSuccess().getValue())) {
+            ResponseStatus.Builder reponseStatusBuilder = ResponseStatus.newBuilder();
+            safeSetProtoField(reponseStatusBuilder::setStatus, SUCCESS);
+            safeSetProtoField(responseBuilder::setResponseStatus, reponseStatusBuilder);
+            safeSetProtoField(responseBuilder::setHotelCode,response.getHotelRates().getHotel().get(0).getID());
+            List<Rate>rateList=response.getHotelRates().getHotel().get(0)
```

Comment:
Please give a a space after List<Rate> and rateList. Please give space before and after =